examples/LambdaExample/.gitignore

```diff
@@ -0,0 +1,342 @@
+# CDK asset staging directory
+.cdk.staging
+cdk.out
+
```

Comment:
holy cow this is a huge .gitignore :)

it's not a huge deal, this is probably fine, but I think I might empty this out and then just add in the stuff that you actually need as you see what shows up in the git status list.  Probably the vast majority of this we don't need and it might be confusing to people, or hard to reason about.